studiopress/simple-social-icons
# Add WhatsApp and Meetup icons

Adds WhatsApp and Meetup icons.

<img width="101" alt="Screen Shot 2022-07-27 at 8 58 13 AM" src="https://user-images.githubusercontent.com/67485942/181268170-2e7daf6b-bdf1-467f-858a-1a1170b150c6.png">




### How to test

1. Open the Simple Social Icons plugin on this branch
2. Add links to the form fields for WhatsApp and Meetup
3. Check that the icons display correctly and they link to the correct place

### Documentation




### Suggested changelog entry

- Add WhatsApp and Meetup icons

### Notes
